### PR TITLE
NPM Build Fixes

### PR DESCRIPTION
**Pull request by andyfreer**
Fix for op.buffer2Insert error & improvements npm build

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,10 +4,11 @@
   "description": "x11 javascript hashing algorithm in pure javascript",
   "main": "index.js",
   "scripts": {
-    "test": "mocha test/test.js",
+    "test": "mocha --timeout 10000 --full-trace --check-leaks test/test.js",
     "clean": "rm -rfv dist/*",
     "benchmark": "node test/benchmarks.js",
-    "build": "grunt"
+    "lint": "node_modules/.bin/eslint lib",
```
Comment (nmarley) [inline package.json]: Why not just `eslint lib`? We shouldn't need to specify path with `node_modules/.bin/` in the scripts...

File: package.json [hunk 2]
```diff
@@ -4,10 +4,11 @@
   "description": "x11 javascript hashing algorithm in pure javascript",
   "main": "index.js",
   "scripts": {
-    "test": "mocha test/test.js",
+    "test": "mocha --timeout 10000 --full-trace --check-leaks test/test.js",
```
Comment (nmarley) [inline package.json]: @andyfreer In the future I would prefer if we split this into another PR as this isn't directly related to the .eslint changes (per the [contribution guidelines written in Dash Core](https://github.com/dashpay/dash/blob/master/CONTRIBUTING.md#pull-request-philosophy)).

Comment (andyfreer) [reply package.json]: ok

File: package.json (hunk 1 above)
Comment (andyfreer) [reply package.json]: fixed

**Pull request merged**